driver/src/contracts/base_contract.rs

```diff
@@ -1,10 +1,18 @@
+extern crate ethereum_tx_sign;
```

Comment:
If I am not mistaken, `extern crate` statements are no longer needed in edition 2018 which I believe is being used.

Reply:
oops, looks like the comment got added twice?